packages/babel-plugin-transform-for-of/test/fixtures/assumption-iterableIsArray/for-of-modify-iterable/input.js

```diff
@@ -0,0 +1,3 @@
+for (let o of arr) {
+  arr = null;
+}
```

Comment:
Is this test already passing in the main branch? I will be surprised if the `arr` binding is constant given then reassignment.